lib/teslamate/vehicles/vehicle.ex

```diff
@@ -28,11 +28,24 @@ defmodule TeslaMate.Vehicles.Vehicle do
   end
 
   @asleep_interval 30
-  @driving_interval 2.5
 
   @drive_timeout_min 15
 
   # Static
+  def interval(env_var, default) do
+    System.get_env(env_var)
+    |> case do
+      nil -> default
+      interval -> String.to_integer(interval)
+    end
+  end
+
+  def asleep_interval, do: interval("POLLING_ASLEEP_INTERVAL", @asleep_interval)
```

Comment:
My vote would be to not let the user set intervals via env variables shorter than our defaults.

Reply:
Good idea, it acts as a safeguard for those who would like to play with what they do not master. I'll implement it.

Reply:
It also means we can increase the defaults later on, if deemed necessarily, and people will get the increased limits even if they have smaller values.

Maybe the limits should be displayed in the website. So the user knows what they got (otherwise I can just imagine bug reports - I changed the interval to 1 second but it doesn't seem to have changed anything!). But may be not worry about that just, that is starting to sound like feature creep for  what should be a simple pull request. Instead, create a new issue if this is merged.